src/ScrollContainer.tsx

```diff
@@ -3,6 +3,7 @@ import { ScrollContainerContext } from "./ScrollContext";
 import environment from "./utils/environment";
 
 interface IProps {
+  snap: string;
```

Comment:
What about set specific type
like,
```suggestion
  snap: 'none' | 'mandatory' | 'proximity';
```

but you don't have to. When I can afford, I'll do this and merge it after test.
Appreciate it for PR with new feature.

Reply:
Yes, of course. I can go back and make this change in a little bit (my first typescript code so I’m still new with the types formatting being added to the JS code).

Reply:
Just pushed this and resolved with 8e92193.